roles/ceph-defaults/defaults/main.yml

```diff
@@ -452,6 +452,9 @@ os_tuning_params:
   - { name: vm.swappiness, value: 10 }
   - { name: vm.min_free_kbytes, value: "{{ vm_min_free_kbytes }}" }
 
+# For Debian & Red Hat/CentOS installs set TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES
+#ceph_tcmalloc_max_total_thread_cache: 134217728
```

Comment:
Please uncomment it otherwise the tasks above will fail.

Reply:
d'oh - of course, initially i had wanted to use the package defaults - which arent necessarily 128MB - will fix.